pkg/compiler/codegen.go

```diff
@@ -1455,6 +1455,14 @@ func (c *codegen) convertBuiltin(expr *ast.CallExpr) {
 		emit.Int(c.prog.BinWriter, 3)
 		emit.Opcode(c.prog.BinWriter, opcode.ROLL)
 		emit.Opcode(c.prog.BinWriter, opcode.MIN)
+		if !c.scope.voidCalls[expr] {
+			// insert top item to the bottom of MEMCPY args, so that it is left on stack
+			emit.Opcode(c.prog.BinWriter, opcode.DUP)
+			emit.Int(c.prog.BinWriter, 6)
+			emit.Opcode(c.prog.BinWriter, opcode.REVERSEN)
+			emit.Int(c.prog.BinWriter, 5)
+			emit.Opcode(c.prog.BinWriter, opcode.REVERSEN)
```

Comment:
Meh... Though I don't see how we can optimize it, creating another local variable is not an option.

Reply:
Neo 2 VM had `XTUCK` for this.